### DISCUSSION TITLE
Module not found: Can't resolve 'base64-js'

### DISCUSSION THREAD

**Discussion by chaudharykiran** [vercel/next.js#47813] [Help]
### Summary

I am using latest nextjs

"next": "latest",
"react": "18.2.0",
 "react-dom": "18.2.0"
 
 I am trying to use Image component like below
 ```
 <Image
        src="/images/profile.jpg" // Route of the image file
        height={144} // Desired size with correct aspect ratio
        width={144} // Desired size with correct aspect ratio
        alt="Your Name"
      />
 ```
     
I am getting below error 

../../../../node_modules/buffer/index.js:11:0
Module not found: Can't resolve 'base64-js'

      
 Any solution to solve such issue?

**Comment by albertwh1te** [comment]
any progess?
